### PR TITLE
Added student details

### PR DESCRIPTION
**Pull request by maitreyav**
Added student details

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://maitreyav.github.io/gci15.fossasia.org

**Comment by samarsault** [comment]
Please resolve the merge conflicts. Also, The image must be square i.e. width = height.

**Comment by maitreyav** [comment]
I've updated the photo dimensions.

**Comment by maitreyav** [comment]
Please give me the write access @samarjeet27

File: _data/students.yml [hunk 1]
```diff
@@ -314,6 +314,15 @@
   twitter: hannniieeeee
   bio: Singaporean coder and classical music enthusiast!
 
+<<<<<<< HEAD
```
Comment (samarsault) [inline _data/students.yml:326]: Please remove git's info.

**Comment by samarsault** [comment]
Also, please squash your commits into one.

File: _data/students.yml (hunk 1 above)
Comment (maitreyav) [inline _data/students.yml:326]: Done.

**Comment by maitreyav** [comment]
How to squash commits into one? @samarjeet27

**Comment by samarsault** [comment]
See this for info https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by mananwason** [comment]
@maitreyav you can also refer [this](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md). Any doubts feel free to ask.

**Comment by maitreyav** [comment]
Hi @mananwason, Thanks for the link. I tried the  git rebase command to squash my commits but it doesn't seems to work what could be wrong? 

`git rebase -i HEAD~7` 

Cannot rebase: You have unstaged changes.
Please commit or stash them.

**Comment by mananwason** [comment]
You have done some changes after the last commit. If they aren't relevant
for the codebase, you can do `git reset --hard HEAD`

**Comment by maitreyav** [comment]
@mananwason  I've force push my commits after squashing them into one.  What should I do next to finish the PR?

 `git push -f origin gh-pages`

Everything up-to-date

**Pull request closed without merging**